<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="AdminisorsAir62.lan" name="Comparision suite" tests="2" failures="2" timestamp="7 May 2017 07:09:23 GMT" time="1.651" errors="0">
  <testcase name="testOne" time="1.583" classname="test.TestClass4">
    <failure type="junit.framework.AssertionFailedError" message="expected:&amp;lt;4&amp;gt; but was:&amp;lt;5&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: expected:<4> but was:<5>
	at junit.framework.Assert.fail(Assert.java:50)
	at junit.framework.Assert.failNotEquals(Assert.java:287)
	at junit.framework.Assert.assertEquals(Assert.java:67)
	at junit.framework.Assert.assertEquals(Assert.java:199)
	at junit.framework.Assert.assertEquals(Assert.java:205)
	at test.TestClass4.testOne(TestClass4.java:20)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- testOne -->
  <testcase name="testOne" time="1.613" classname="test.TestClass4">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at utils.Helper.takeScreenShot(Helper.java:377)
	at listeners.TestListner.onTestFailure(TestListner.java:27)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- testOne -->
</testsuite> <!-- Comparision suite -->
